downloadmanager/src/main/java/cm/aptoide/pt/downloadmanager/RetryFileDownloadManager.java

```diff
@@ -0,0 +1,92 @@
+package cm.aptoide.pt.downloadmanager;
+
+import cm.aptoide.pt.logger.Logger;
+import rx.Completable;
+import rx.Observable;
+import rx.Subscription;
+import rx.subjects.PublishSubject;
+
+public class RetryFileDownloadManager implements RetryFileDownloader {
+
+  private static final String TAG = "RetryFileDownloadManage";
```

Comment:
Missing the “r"

Reply:
Tag can only have 21 characters max, so it doesn't fit